anjani/plugins/welcome.py

```diff
@@ -265,7 +265,7 @@ async def cmd_setwelcome(self, ctx: command.Context) -> str:
 
         try:  # Try to build a text first to check message validity
             await self._build_text(welc_text or "", ctx.author, chat, self.bot.client)
-        except KeyError as e:
+        except (KeyError, ValueError) as e:
```

Comment:
buat error message baru aja?
returnnya gini `raised from ValueError: Single '}' encountered in format string`

Reply:
hardcoded gitu? ntr kalo errornya lagi `'{'` kan masa diprintnya `'}'`

Reply:
hmmm 'Failed when parsing text due to unmatched bracked' ?

Reply:
terlalu mengharcoded, kan bisa aja valuerror dari yg lain